<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard - Inkpotro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/inkpotro.png</normaloff>../icons/inkpotro.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="sidebar_frame">
     <layout class="QVBoxLayout" name="sidebar">
      <item>
       <widget class="QLineEdit" name="title_input">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #3c3c3c;
    border-radius: 8px;
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid #428cfe;
}</string>
        </property>
        <property name="placeholderText">
         <string>Post Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date_input">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #3c3c3c;
    border-radius: 8px;
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid #428cfe;
}</string>
        </property>
        <property name="placeholderText">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category_input">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #3c3c3c;
    border-radius: 8px;
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid #428cfe;
}</string>
        </property>
        <property name="placeholderText">
         <string>Category (comma separated)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tags_input">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #3c3c3c;
    border-radius: 8px;
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid #428cfe;
}</string>
        </property>
        <property name="placeholderText">
         <string>Tags (comma separated)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="author_input">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #3c3c3c;
    border-radius: 8px;
    padding: 6px 10px;
}

QLineEdit:focus {
    border: 2px solid #428cfe;
}</string>
        </property>
        <property name="placeholderText">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="draft_checkbox">
        <property name="text">
            <string>Mark as Draft</string>
        </property>
        </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_locally_btn">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: 2px solid #3c3c3c;
    padding: 6px 10px;
    border-radius: 6px;
}

QPushButton:pressed {
    background-color: #428cfe;
	color: #e7ecfb;
	padding-left: 10px;
    padding-top: 9px;
}</string>
        </property>
        <property name="text">
         <string>Save Locally</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="editor_tab">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_editor">
       <item>
        <widget class="QTextEdit" name="editor_textedit">
         <property name="styleSheet">
          <string notr="true">QTextEdit {
    border: 2px solid #3c3c3c;
    border-radius: 8px;
    padding: 6px 10px;
}

QTextEdit:focus {
    border: 2px solid #428cfe;
}</string>
         </property>
         <property name="placeholderText">
          <string>Write your markdown content here...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="preview_tab">
      <attribute name="title">
       <string>Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_preview">
       <item>
        <widget class="QTextBrowser" name="preview_browser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
